src/markdown-pages/automate-workflows/get-started-new-relic-cli.mdx

```diff
@@ -32,42 +32,42 @@ Access the New Relic platform from the comfort of your terminal: you can use the
 
 For this guide you just need:
 
-1. Your New Relic [personal API Key](https://docs.newrelic.com/docs/apis/get-started/intro-apis/types-new-relic-api-keys#personal-api-key), which you can create from the **Account settings** of your New Relic account
-2. An [instrumented application](https://docs.newrelic.com/docs/agents/manage-apm-agents/installation/install-agent) in your New Relic account
+- Your New Relic [personal API Key](https://docs.newrelic.com/docs/apis/get-started/intro-apis/types-new-relic-api-keys#personal-api-key), which you can create from the **Account settings** of your New Relic account
+- An [instrumented application](https://docs.newrelic.com/docs/agents/manage-apm-agents/installation/install-agent) in your New Relic account
 
 <Steps>
 <Step>
 
 ## Install the New Relic CLI
 
-The New Relic CLI can be downloaded via [Homebrew](https://brew.sh/) (macOS), [Scoop](https://scoop.sh/) (Windows), and [Snapcraft](https://snapcraft.io/) (Linux). You can also download [pre-built binaries](https://github.com/newrelic/newrelic-cli/releases) for all platforms, including .deb and .rpm packages, and our Windows x64 .msi installer.
+Download the New Relic CLI for your operating system, as described below. You can also download [pre-built binaries](https://github.com/newrelic/newrelic-cli/releases) for all platforms, including .deb and .rpm packages, and our Windows x64 .msi installer.
 
-### Linux
+**Linux**
```

Comment:
Some of these changes are different than those from the edit pass for [Quickly tag a set of resources](https://github.com/newrelic/developer-website/pull/733), which has a nearly identical section. I'm not going to make those changes to this PR, but I'd like to consolidate styles across the site over time, where it makes sense.

In this case, I'd argue that linking to the repo, which contains more detailed [instructions](https://github.com/newrelic/newrelic-cli#installation), is better and more manageable than writing these instructions in multiple articles on the dev site.